Composer/plugins/localPublish/src/index.ts

```diff
@@ -200,7 +201,7 @@ class LocalPublisher {
     const commandAndArgs =
       settings.runtime && settings.runtime.customRuntime === true
         ? settings.runtime.command.split(/\s+/)
-        : ['dotnet', 'run'];
+        : ['dotnet', 'run', '--project', 'azurewebapp']; //TODO: ccastro should pick up the bot start command here. After, remove azurewebapp arg
```

Comment:
is this a todo that can be done now?

Reply:
I think it should be in scope, yes. I'll try that later!